Traceback (most recent call last):
  File "snippet.py", line 27, in <module>
    import svn.fs
ImportError: No module named fs
